app/models/event.rb

```diff
@@ -82,4 +82,10 @@ def can_leave?(user)
   def standing_of(team)
     'Gewinner ' + team.id.to_s
   end
+
+  def add_test_teams
```

Comment:
NEIN!
Das gehört doch in eine Test-Datei? Zumindest nicht in das Event-Model.

Reply:
Oh yes, about that method... :D

It's bad, I know. Nobody should use the factory bot in production code. It also produces weird errors when re-starting the server (with events already in the database) and creating a new event.

But: it is really helpful. I do not want to create 16 fake email accounts and register 16 times on the platform to test the creation of event schedules by hand. And it was already there, I am not the original author of this method.

Now that I think of it, we could try to find a better place for calling this method.